js/apps/system/_admin/aardvark/APP/react/public/index.html

```diff
@@ -11,6 +11,15 @@
     <meta name="author" content="Heiko Kernbach, Michael Hackstein">
     <script src="%REACT_APP_ARANGODB_CONFIG_PATH%"></script>
   </head>
+
+  <script>
```

Comment:
Why is  this necessary?

Reply:
First of all, this is only for development. I could not change the default behaviour for the react dev mode. That script tag is required to "link" to our UI default route. So basically if you start the react dev mode server (`npm start`) this will setup a http server (port 3000), start a browser, go to `/`and link then to the correct starting point.